Backends/CLX/medium.lisp

```diff
@@ -92,16 +92,37 @@ region and its clipping pixmap. This is looked up for optimization with region-e
           (warn "Unknown cap style ~S, using :round" clim-shape)
           :round))))
 
+;;; XXX: this should be refactored into a reusable protocol in clim-backend
+;;; with specialization on medium. -- jd 2018-10-31
+(defun line-style-scale (line-style medium)
+  (let ((unit (line-style-unit line-style)))
+    (ecase unit
+      (:normal 1)
+      (:point (/ (graft-width (graft medium))
+                 (graft-width (graft medium) :units :inches)
+                 72))
+      (:coordinate (multiple-value-call #'(lambda (x y) (sqrt (+ (expt x 2) (expt y 2))))
```

Comment:
Why not `multiple-value-bind`?

Reply:
at that point of time it seemed more natural (I didn't even think about m-v-b), but when you mention it I agree that it m-v-b is more idiomatic here. Fixed.